pkg/products/rhssocommon/reconciler_test.go

```diff
@@ -46,7 +48,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/record"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
-	//fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
```

Comment:
Lets remove this instead of commenting out

Reply:
Done. There couple of commented imports as well. https://github.com/makslion/integreatly-operator/blob/MGDAPI-1253/pkg/products/rhssocommon/reconciler_test.go#L20-L29 They also to be removed manually? :thinking:

Reply:
Yes, I think they can be removed also. They can later be re-imported if they end up being used again :thinking:

Reply:
Removed all unused imports from changed files